README.md

```diff
@@ -4,9 +4,10 @@ An Ansible role for installing Docker.
 
 ## Role Variables
 
-- `docker_repository_arch` - Arch version of docker package (default `amd64`)
-- `docker_package_name` - Name of docker package (`docker-ce` or `docker-ee`, defaults to `docker-ce`)
-- `docker_version` - Docker version (default `17.*`)
+- `docker_repository_arch` - Architecture for Docker package repository (default: `amd64`)
+- `docker_packages` - Names of Docker packages (default: `docker-ce`, `docker-ce-cli`, `containerd.io`)
```

Comment:
The defaults here don't align with how this variable is actually used. Folks would have to look at the source code to know that the format has to be `package=version`.

Reply:
Addressed in d723c514f5b48fa04cac6aeb7a04d8247573b57a.